src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -271,7 +270,7 @@ class FilterButton extends React.Component<Props, State> {
                                             columns={columns}
                                             selectedConnector={selectedConnector}
                                             update={this.update}
-                                            updateselectedConnector={this.updateselectedConnector}
+                                            onConnectorChange={this.onConnectorChange}
```

Comment:
This might be a matter of preference. The convention I've observed is...
1. The left-side prop name uses the "on" format
2. The right-side function uses the "handle" format

It is analogous to the event name and event handler.